docs/server/source/appendices/log-rotation.md

```diff
@@ -0,0 +1,46 @@
+Copyright BigchainDB GmbH and BigchainDB contributors
+SPDX-License-Identifier: (Apache-2.0 AND CC-BY-4.0)
+Code is Apache-2.0 and docs are CC-BY-4.0
+--->
+
+# Log rotation for a BigchainDB node
```

Comment:
This is a new docs page so it must be added to an `index.rst` file otherwise Sphinx will say, for example:
```bash
Warning, treated as error:
/home/troy/repos/bigchaindb/docs/server/source/appendices/log-rotation.md:document isn't included in any toctree
```

It's in the appendices directory, so that's the `index.rst` file that should be updated, i.e.

`docs/server/source/appendices/index.rst`

Reply:
forgot to add it will add in next commit.